lib/assessment.rb

```diff
@@ -13,28 +13,32 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    reduce (memo, f) { return memo + f }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
+
+  # Yeah... confused
   def some_odd(numbers)
-    # Your code goes here
+    each_with_index { |number, index| number.odd? }
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
+
+  # same comment as above
   def every_even(numbers)
-    # Your code goes here
+    each_with_index { |number, index|  number.even? }
```

Comment:
See above; `even?` is correct, but your enumerable method is wrong. Which method tells us if _every_ element in a particular set passes some test?